tsconfig.json

```diff
@@ -25,7 +25,7 @@
   },
   "include": [
     "src/**/*"
-  ],
+, "pages/helpers.ts"  ],
```

Comment:
So funky thing about Next.js is that treats everything in the `pages` directory as a route. So you can go to `localhost:3000/helpers` and it'll actually attempt to render a page from that 😱
 
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/19421190/140235821-ccbca86a-59dc-4a2e-999a-0d47138d5ea9.png">

This is all to say, we should put this in a separate folder. I'm thinking a top-level `helpers` right at the root should be fine.

Reply:
Moved, imports refactored, and pushed!